openapi/p4pa-fileshare.openapi.yaml

```
@@ -115,6 +115,54 @@ paths:
                 format: binary
         '404':
           description: Not Found
+  /organization/{organizationId}/uploadsendfiles/{sendNotificationId}:
```

the upload will be suggested by the method and the operationId
let's avoid to use upload on the path: if we will expose the download, the path would be the same, but it will change the method
```suggestion
  /organization/{organizationId}/send-files/{sendNotificationId}:
```